### DISCUSSION TITLE
Security concern

### DISCUSSION THREAD

**Discussion by quaso** [ente-io/ente#2299] [General]
I ran self-hosted Ente (with minio) exposed port 8080 to internet. I was able to connect Ente mobile app to my local server. However, when app was running, my local firewall noticed some suspicious outgoing communication from `/goform/set_LimitClient_cfg` to some unkown IP remote 45.128.232.200:55302. I was not expecting that and immediately stopped self hosted Ento and will never run it again until some clarification. 

What is the reason to communicate with any remote address while using self hosted Ento?

**Comment by quaso** [comment]
Forgot to notice, I am using official docker compose yaml with image ghcr.io/ente-io/server

**Comment by mnvr** [comment]
It sounds like an incoming request **to** /goform/set_LimitClient_cfg, not **from** the app. if you expose servers on the public internet, invariably you'll get incoming requests from people seeking for vulnerabilities, it has nothing to do with the app (e.g. try running any web server, you'll notice similar incoming requests like this)

**Comment by quaso** [comment]
The firewall clearly states it is an outgoing traffic, I triple checked
that. I have other web server exposed using other port and there was no
firewall alert.

**Comment by quaso** [comment]
![image](https://github.com/ente-io/ente/assets/8343174/55eb9276-9561-480a-9e26-412acedae28d)
95.102.115.49 is my external dynamic IP

**Reply by quaso** [reply]
Looking at that for x-th time, I think you are correct. The external IP is "attacking" internal one. So somebody from public internet initialized the attack. No issue with Ente.
